src/components/JobPost/index.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import { Form, Col, Row, Container, Button, Image } from 'react-bootstrap'
+import uploadImage from '../../media/upload-Image.png'
+import DatePicker from './DatePicker'
+
+const JopPost = () => {
```

Comment:
Jobpost, with b. You already have this same form in the CompanyProfile function.